### ISSUE TITLE
Labels for training

### ISSUE THREAD

**Issue by SoumiDas** [gwwang16/Lyft-Perception-Challenge#1]
Hi,

I just wanted to clarify 1 point i.e are the labels for training the semantic segmentation model for CARLA the black images where the class information is provided in the red channel? or is it the converted image where different classes get different colors?
It'd be really helpful to know!

Thanks!

**Comment by gwwang16** [comment]
Yes, the class information is in the red channel.  You can find these class names in readme or `img_process.py`.
